mantidproject/mantid
# Disable file logging & simplfy log filtering

**Description of work.**
Removed file logging, and changed log filtering so you cannot set separate log filters for each log (few people understood this anyway).

**To test:**
1. Check unit and system tests work
1. Start mantiplot, play with the log levels in the message display, check they still work.
1. Check that the message level is remembered when closing and restarting Mantid.

Fixes #23102 


Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
